### ISSUE TITLE
Implement preferences at the 'drawer' level via the opensocial gadget api

### ISSUE THREAD

**Issue by trevershick** [trevershick/jook#8]
```
 <?xml version="1.0" encoding="UTF-8" ?>
 <Module>
   <ModulePrefs 
     title="List Data Type Example" 
     scrolling="true"/> 
   <UserPref name="mylist" 
     display_name="Add Terms" 
     datatype="list" />
```

[see here](http://code.google.com/apis/gadgets/docs/fundamentals.html#URL)

**Comment by trevershick** [comment]
Static interactions appear to CLOSELY resemble open social gadgets. I believe that we could easily implement the open social gadget API in jook.  OR implement the jook container on top of Apache Shindig

**Comment by trevershick** [comment]
First thing to do is expose a json based service for managing one's own preferences.

**Comment by trevershick** [comment]
Closed by 66c3903675f7d51bba885afea77e4d3fc3f6bd0b.
Closed by 66c3903675f7d51bba885afea77e4d3fc3f6bd0b.
